src/main/java/screens/GraphInfo.java

```diff
@@ -0,0 +1,123 @@
+package screens;
+
+/**
+ * Class that can store information about the current state on the drawn graph used for navigation.
+ */
+public class GraphInfo {
+
+    /**
+     * The radius of the currently drawn graph.
+     */
+    private int currentRadius;
+
+    /**
+     * The center node id of the currently drawn graph.
+     */
+    private int currentCenterNode;
+
+    /**
+     * The number of genomes paths that are specified.
+     */
+    private double genomeNum;
+
+    /**
+     * All specified genome paths per node.
+     */
+    private int[][] paths;
+
+    /**
+     * A singleton instance of GraphInfo.
+     */
+    private static GraphInfo instance;
+
+    /**
+     * Constructor for the information.
+     */
+    public GraphInfo() {
+        this.currentCenterNode = 0;
```

Comment:
Is there a reason these values are the default? I believe that currently when loading a file, the default is something else.